src/app/services/species.service.spec.ts

```diff
@@ -10,8 +10,7 @@ describe('SpeciesService', () => {
 
   beforeEach(() => {
     TestBed.configureTestingModule({
-      imports: [HttpClientModule],
-      providers: [HttpClient],
+      imports: [HttpClientModule]
```

Comment:
Why are we using this module instead of the testing module?

https://github.com/gSchool/ng-rickdiculous-app/blob/solution/src/app/services/episodes.service.spec.ts

Reply:
We have corrected this.